js/src/viewer/manifestListItem.js

```diff
@@ -45,6 +46,9 @@
       });
 
       this.fetchTplData(this.manifestId);
+      if (this.forcedIndex !== null) {
```

Comment:
This is an adaptation I made to allow the collection tree to take over the ordering of manifest items by forcing its "index" number to be a specified value. The current implementation of manifestListItem.js orders by when it was loaded into the global state, which is no longer a good indicator with the way manifests are loaded under collections.